tests/acceptance/config/behat-core.yml

```diff
@@ -304,17 +304,6 @@ default:
         - WebDavLockingContext:
         - WebDavPropertiesContext:
 
-    coreApiWebdavLocks3:
```

Comment:
why removing the whole suite? I see you have deleted one file in that folder, but others remain

Reply:
All feature files from suite `coreApiWebdavLocks2` have been removed, so suite `coreApiWebdavLocks3` features have been moved to `coreApiWebdavLocks2`